src/lib/fragments/AgGrid.react.js

```diff
@@ -126,6 +129,25 @@ export default class DashAgGrid extends Component {
         };
 
         this.selectionEventFired = false;
+        this.reference = React.createRef();
+    }
+
+    onPaginationChanged() {
+        const {setProps} = this.props;
+        const {gridApi} = this.state;
+        if (gridApi) {
+            setProps({
+                paginationInfo: {
+                    paginationIsLastPageFound:
+                        gridApi.paginationIsLastPageFound(),
+                    paginationGetPageSize: gridApi.paginationGetPageSize(),
+                    paginationGetCurrentPage:
+                        gridApi.paginationGetCurrentPage(),
+                    paginationGetTotalPages: gridApi.paginationGetTotalPages(),
+                    paginationGetRowCount: gridApi.paginationGetRowCount(),
```

Comment:
Rather than matching the method names exactly, let's use the obvious names you'd give the results of these methods - ie strip off `paginationGet` so each is a noun. And then so it's fully self-documenting, let's describe them in `propTypes`, ie `paginationInfo: PropTypes.exact({...})`
```suggestion
                    isLastPageFound: gridApi.paginationIsLastPageFound(),
                    pageSize: gridApi.paginationGetPageSize(),
                    currentPage: gridApi.paginationGetCurrentPage(),
                    totalPages: gridApi.paginationGetTotalPages(),
                    rowCount: gridApi.paginationGetRowCount(),
```